addons/edi/models/edi_document.py

```diff
@@ -605,6 +605,34 @@ def action_view_outputs(self):
         }
         return action
 
+    @api.model
+    def action_execute_transferred_docs(self):
+        """
+        EDI documents transferred could be in 'new' or 'prep' state depending on the
+        `processing_level` of the document type. Process these documents later using this method,
+        usually triggered by a cron job. Collect documents to process that are:
+            * created by edi transfer
+            * doc type processing level is 'disabled' and doc is in 'draft' state
+            * doc type processing level is 'prepare' and doc is in 'prep' state
+        """
+        docs_to_process = self.search(
+            [
+                "|",
+                "&",
+                ("doc_type_id.processing_level", "=", "disabled"),
+                ("state", "=", "draft"),
+                "&",
```

Comment:
It looks like there is an "|" or "&" needed somewhere, sure that is working as expected?

Question: Can you check please if transfer_id is better to be in the search or to filter after?

Reply:
With a complex domain like this i'd recommend using `from odoo.osv import expression` and `expression.AND()`, `expression.OR()`